source/clear-linux/reference/collaboration/community/conduct.rst

```diff
@@ -0,0 +1,98 @@
+**Collaborate openly.** Collaboration is central to the |CL| project and to
+the larger free software community. This collaboration involves individuals
+working within teams and working with other projects outside of the |CL|
+community. This collaboration reduces redundancy and improves the quality of
+our work. Internally and externally, we should always be open to
+collaboration. Wherever possible, we should work closely with upstream and
+downstream projects and others in the free software community to coordinate
+our technical, advocacy, documentation, and other work. Our work should be
+done transparently and we should involve as many interested parties as early
+as possible. If we decide to take a different approach, we will let them know
+early, document our work, and inform others regularly of our progress. Do not
+create private forms of communication that take away transparency or exclude
+other contributors and collaborators.
+
+**When we disagree, try to understand why.** Disagreements, both social and
+technical, happen all the time and the |CL| community is no exception.
+Resolve disagreements and differing views constructively. Remember, we are
+different. The strength of the community comes from its diversity: people
+from a wide range of backgrounds. Different people have different
+perspectives on issues. Being unable to understand why someone holds a
+viewpoint does not mean they are wrong. Focus on helping resolve issues and
+learning from mistakes. Resolve disagreements and differing views
+constructively and with the help of the community and community processes.
+
+**When we are unsure, we ask for help.** Nobody knows everything and nobody
+is expected to be perfect in the |CL| community. Asking questions avoids many
+problems down the road. So, questions are encouraged. If you are asked
+questions, be responsive and helpful. However, when asking a question, take
+care to do it in an appropriate forum such as the `mailing list`_ or the
+#clearlinux `IRC channel`_.
+
+**Be careful with your words and actions.** We are a community of
+professionals and we conduct ourselves professionally. Do not insult or put
+down other participants. Harassment and other exclusionary behavior is not
+acceptable and should be reported. This includes but is not limited to:
+
+* Violent threats or language directed against another person.
+* Discriminatory jokes and language.
+* Posting sexually suggestive, explicit, or violent material.
+* Posting, or threatening to post, other people's personally identifying
+  information, AKA "doxing".
+* Personal insults, especially those using racist or sexist terms.
+* Unwelcome sexual attention.
+* Advocating for, or encouraging, any of the above behavior.
+* Repeated harassment of others. In general, if someone asks you to stop,
+  then stop.
+
+.. note::
+   Portions of the |CL| code of conduct are derived from the code of conduct
+   of OpenStack\*, the PyCon Conference, and the Django\* Project.
```

Comment:
This code of conduct is missing enforcement/responsibility. Instead of taking from these three projects, we could we use the contributor covenant code of conduct (https://www.contributor-covenant.org) as it is well established for open source projects. It has >15,000 adopters incl. GitLab, K8, and Jenkins.

Reply:
This note's intent is to provide further context by providing the sources used to create the code of conduct. There is no enforcement section since we don't have any process to handle complaints. Who should be contacted or what penalties would apply? Both questions for which I have no answer for this first version. In the near future they should be addressed though.